.docs/Add-VSTeamVariableGroup.md

```diff
@@ -0,0 +1,138 @@
+<!-- #include "./common/header.md" -->
+
+# Add-VSTeamVariableGroup
+
+## SYNOPSIS
+<!-- #include "./synopsis/Add-VSTeamServiceEndpoint.md" -->
+
+## SYNTAX
+
+```powershell
+Add-VSTeamVariableGroup [-variableGroupName] <String> [-variableGroupType] <String>
+ [-variableGroupDescription] <String> [-variableGroupVariables] <Hashtable>
+ [[-variableGroupProviderData] <Hashtable>] [-ProjectName] <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+<!-- #include "./synopsis/Add-VSTeamServiceEndpoint.md" -->
+
+## EXAMPLES
+
+## PARAMETERS
+
+### -ProjectName
+The name of the project. 
+You can tab complete from the projects in your Team Services or TFS account when passed on the command line.
+
+```yaml
+Type: String
+Parameter Sets: (All)

```

Comment:
You don't need to add (All) to each parameter when you only have a default parameter set for the function.